docs/sources/tempo/getting-started/tempo-in-grafana.md

```diff
@@ -53,32 +53,13 @@ For more information, refer to the [service graph view]({{< relref "../metrics-g
 
 <p align="center"><img src="../assets/apm-overview.png" alt="Service graph view overview"></p>
 
-## Metrics from spans
-
-RED metrics can be used to drive service graphs and other ready-to-go visualizations of your span data. RED metrics represent:
-
-- Rate, the number of requests per second
-- Errors, the number of those requests that are failing
-- Duration, the amount of time those requests take
-
-For more information about RED method, refer to [The RED Method: How to instrument your services](/blog/2018/08/02/the-red-method-how-to-instrument-your-services/).
```

Comment:
RED metrics are mentioned on tempo-in-grafana.md, api_docs/metrics-summary.md, metrics-generator/_index.md, metrics-generator/service-graph-view.md, operations/monitoring.md, and the v2.2.md release notes. However, this explanation is the most detailed and the only one that links out to other resources. 

What would you think about including this explanation some place else? Maybe on the metrics-generator/_index.md underneath "Span metrics"?  The first sentence in that section says "The span metrics processor derives RED (Request, Error and Duration) metrics from spans." That could be followed with every the second sentence from the above removed content from lines 58-64 (from "RED Metrics represent..."). 

What do you think?

Reply:
> I came up with another idea (original is below). What do you think about creating a new page in the getting-started folder called "Metrics from traces". This page would have the content from both of "Metrics from traces" and "Linking traces and metrics." That lets us move the content out of this page but still highlight the capabilities. This would let us feature the metrics-generator in the getting started section and focus the Tempo in Grafana page a little more.

I think that's a good idea. I initially considered making a new page under [metrics-generator](https://grafana.com/docs/tempo/latest/metrics-generator/), though, what you're proposing seems to be more appropriate for what we're highlighting in the content.

> I can help with [creating a new page](https://grafana.com/docs/writers-toolkit/structure/topic-types/concept/), too, if you like the idea.

I think I'll be fine to create the new page, the docs do a great job of explaining how; the other pages also serves as great examples!

Reply:
Great minds think alike! I also looked at a page under metrics-generator, but then realized we also have docs under Configure > Grafana Agent that talk about [span metrics](https://grafana.com/docs/tempo/latest/configuration/grafana-agent/span-metrics/). Introducing the concepts earlier can help readers understand the feature Tempo offers.

Reply:
> Introducing the concepts earlier can help readers understand the feature Tempo offers.

Agreed. I also think separating these concepts in Getting Started can help reader retention since there's less content to digest at once.

Reply:
Thank you (again) for tackling this. This PR has been a pleasure to work on with you. :)

Reply:
Likewise, it's been a pleasure!